packages/backend/src/services/podman-service.spec.ts

```diff
@@ -334,3 +344,52 @@ describe('isMachineRootful', () => {
     expect(result).toBeFalsy();
   });
 });
+
+describe('podman connections', () => {
+  test('PodmanService#getPodmanConnections should use podman#exec', async () => {
+    vi.mocked(podmanExtensionApiMock.exports.exec).mockResolvedValue({
+      stdout: JSON.stringify([WSL_CONNECTION_INFO_MOCK]),
+      stderr: '',
+      command: 'dummy-command',
+    });
+
+    const podman = getPodmanService();
+
+    const connections = await podman.getPodmanConnections();
+    expect(connections).toHaveLength(1);
+
+    expect(connections[0]).toStrictEqual(WSL_CONNECTION_INFO_MOCK);
+  });
+
+  test('PodmanService#getPodmanConnections should exclude non-ssh connections', async () => {
+    vi.mocked(podmanExtensionApiMock.exports.exec).mockResolvedValue({
+      stdout: JSON.stringify([
+        {
+          ...WSL_CONNECTION_INFO_MOCK,
+          URI: 'localhost:8888',
```

Comment:
```suggestion
          URI: 'tcp://localhost:888',
```

Yes, good idea @gastoner !